package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@helpscout/hsds-react",
-  "version": "3.14.0",
+  "version": "3.14.1-color-contrast-1",
```

Comment:
is this an acceptable version name for the HSDS project?

Reply:
I used it for local testing with beacon & hsapp, according to [this doc](https://helpscout.gitbook.io/hsds-react/testing/local-integration-testing). Should I roll it back now that I feel like PR is ready? Not sure what's the process here 😅

Reply:
lol you and me both 😁 . Wonder if @plbabin or @tinkertrain can chime in with direction?

Reply:
Hi! @maria-p , @luketlancaster

I think that at the moment of release the tool we use (np) sets the correct value there automatically, so there's no need to worry about as far as I know (I've never set my own name but something similar happens when releasing a beta version).